mobile/src/main/java/rs/readahead/washington/mobile/views/activity/CollectFormEntryActivity.kt

```diff
@@ -986,6 +987,15 @@ class CollectFormEntryActivity : MetadataActivity(), ICollectEntryInterface,
         window.clearFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
     }
 
+    override fun hideBottomNavigation() {
+        binding.btmNavMain.visibility = View.GONE
```

Comment:
We could use extension binding.btmNavMain.hide() here and in other similar places.